wazo_confd/plugins/user_blocklist/api.yml

```diff
@@ -97,6 +97,88 @@ paths:
           $ref: '#/responses/UpdateError'
         '404':
           $ref: '#/responses/NotFoundError'
+  /users/{user_uuid}/blocklist/numbers:
+    parameters:
+      - $ref: '#/parameters/useruuid'
+    head:
+      operationId: lookup_user_blocklist_number
+      summary: Check if a number is part of a user's blocklist
+      description: '**Required ACL:** `confd.users.{user_uuid}.blocklist.read`'
+      tags:
+      - blocklist
+      parameters:
+        - $ref: '#/parameters/tenantuuid'
+        - name: number_exact
+          in: query
+          type: string
+          description: E.164 number string to lookup(exact match) in the user's blocklist
+          required: true
+      responses:
+        '204':
+          description: successful match
+          headers:
+            Wazo-Blocklist-Number-Label:
+              type: string
+              description: Label of the blocklisted number
+        '400':
+          $ref: '#/responses/GenericError'
+        '404':
+          $ref: '#/responses/NotFoundError'
+  /blocklist/numbers:
+    get:
+      operationId: list_blocklist_numbers
+      summary: List all blocklisted numbers in tenant
+      description: '**Required ACL:** `confd.blocklist.read`'
+      tags:
+      - blocklist
+      parameters:
+        - $ref: '#/parameters/tenantuuid'
+        - $ref: '#/parameters/recurse'
+        - $ref: '#/parameters/order'
+        - $ref: '#/parameters/direction'
+        - $ref: '#/parameters/limit'
+        - $ref: '#/parameters/offset'
+        - $ref: '#/parameters/search'
+        - name: user_uuids
+          in: query
+          type: string
+          description: comma-separated list of user UUIDs to filter blocklisted numbers
```

Comment:
The list of users is not supported, right?

Reply:
Nope, will change that